src/main/java/com/cloudbees/plugins/credentials/CredentialsProviderFilter.java

```diff
@@ -177,7 +177,7 @@ public String getDisplayName() {
         /**
          * Our constructor.
          *
-         * @param classNames the whitelist of class names.
+         * @param classNames the list of included class names.
```

Comment:
```suggestion
         * @param classNames the list of allowed class names.
```

To keep consistent with the other changes below.